compiler/grainformat/reformat.re

```diff
@@ -0,0 +1,2773 @@
+open Grain;
+open Compile;
+open Grain_parsing;
+open Grain_utils;
+
+module Doc = Res_doc;
+
+let get_raw_pos_info = (pos: Lexing.position) => (
+  pos.pos_fname,
+  pos.pos_lnum,
+  pos.pos_cnum - pos.pos_bol,
+  pos.pos_bol,
+);
+
+let getOriginalCode =
```

Comment:
I think we should try to be consistent with the rest of our Reason codebase and stick to snake_case (for all functions and variables)

Reply:
Fixed. Let me know if you spot any more